docs/modules/user-guide/partials/proc_tagging-and-pushing-a-registry-container-image.adoc

```diff
@@ -0,0 +1,19 @@
+[id="building-a-custom-devfile-registry_{context}"]
+= Tagging and pushing a registry container image
+
+[role="_abstract"]
+Use the devfile index container image to create a container in which you build, run, and deploy your application.
+
+.Procedure
+
+. Consult the documentation for your container registry and log into the documentation with the Docker CLI.
+. Choose a container registry.
```

Comment:
This seems like it would be the first step in this procedure unless I'm missing something obvious. (Which is likely!)

Reply:
I reverted the order. I agree with you, Kevin. 

I made it:

1. Choose a container registry. 
2. Consult the documentation for your container registry and log into the registry with the Docker CLI. 

@johnmcollier would you please confirm this is correct?